src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -163,7 +161,7 @@ private int PostProcess()
             var postProcessor = processorFactory.CreatePostProcessor();
             succeeded = postProcessor.Execute(bootstrapSettings.ChildCmdLineArgs.ToArray(), config, teamBuildSettings);
         }
-
+        logger.AddTelemetryMessage(TelemetryKeys.ScannerEngineAnalysisResult, succeeded ? "success" : "failure");
```

Comment:
This doesn't work. The telemetry is send by the java scanner-cli (or java scanner-engine now), which is called in `postProcessor.Execute`. The telemetry written here will just get lost. The best we can do is writing this property in our java plugin, right before we send the telemetry to the server. `TelemetryKeys.ScannerEngineAnalysisResult` should not be solved in this PR/sprint. It is too complicated.